### ISSUE TITLE
객체 분류 모델 테스트 - Mask R-CNN

### ISSUE THREAD

**Issue by yejinlee99** [pladata-encore/DA36-final-ai-team2-repo#2]
## 🌈 Success criteria

1.  이 이슈를 복사 후 각자 모델을 하나씩 선정해서 이슈 작성 후 테스트 해주세요.

## 👷 To-do

- [x] 모델 불러오기
- [x] 데이터 로컬에 다운로드하기
- [x] 메타 데이터 생성하기
- [x] train / test 데이터 분리하기
- [ ] 모델 학습하기
- [ ] 모델 평가하기
- [ ] 예측하기

## 🧶 Review

- TFRecord를 이용해 Tensorflow로 학습 시도했으나 OOM 이슈 발생, 미해결
- 공식 API 코드 흐름을 따라가며 Pytorch 기반으로 학습 시도
- Roboflow에서 Mask R-CNN 전용 format을 제공하지 않아 yolo용 txt파일로 다운로드한 후 Mask R-CNN COCO-JSON format으로 변환
